app/models/api/hses.rb

```diff
@@ -0,0 +1,26 @@
+require "fake_api_response_wrapper"
+require "fake_issues"
+
+class Api::Hses
```

Comment:
this is here to ensure that staging still works! once we have a working endpoint, we can transition this to our real HTTP request code.